scripts/prettier/prettier-helpers.js

```diff
@@ -67,7 +91,8 @@ function runPrettierForFolder(folderPath, config = {}) {
     folderPath = path.join(repoRoot, folderPath);
   }
 
-  const sourcePath = `"${path.join(folderPath, nonRecursive ? '' : '**', '*')}"`;
+  const fileExtensions = `.{${prettierSupportedFileExtensions.join(',')}}`;
+  const sourcePath = `"${path.join(folderPath, nonRecursive ? '' : '**', '*')}${fileExtensions}"`;
```

Comment:
Q: Am I right to assume that `/mycoolpath/*js,jsx,ts,tsx,...` would be a possible result from this? If so, is it valid?

Reply:
nope 

- either `/mycoolpath/**/*.{js,jsx,ts,tsx}`
- or `/mycoolpath/**/*.{js,jsx,ts,tsx}`

Reply:
we don't have tests for this so I understand it might be not easy to read. we'll get better ;)

Reply:
Then should it be like so?
```suggestion
  const sourcePath = `"${path.join(folderPath, nonRecursive ? '' : '**', '*')}.{${fileExtensions}}"`;
```

Reply:
Disregard, I did not notice that `fileExtensions` already adds the dot and braces 😑